### DISCUSSION TITLE
Backlog

### DISCUSSION THREAD

**Discussion by Siemienik** [exceljs/exceljs#2538] [General]
Hi, I think we should update our backlog and add:
1. Performance
2. Clean up issues
3. Do fixes

For me that are really important things to focus about them. 
What is important for you?

**Comment by IgnatovDan** [comment]
I would prefer Performance: generate [10000 rows x 50 cols] xlsx file in 3 seconds ([see conversation](https://github.com/exceljs/exceljs/issues/742))

**Comment by alubbe** [comment]
I like those three. Right now I'm quite interested in improving the streaming, esp. adding cancellation, can we add that as a fourth?

**Comment by alubbe** [comment]
As a fifth point, I also think we should explore replacing sax (unmaintained for over 2 years) with saxes, which also speeds up our runtime performance (at least for streaming). And I believe @coldhiber also found an issue with the temp file library that uses global state

**Comment by Siemienik** [comment]
So, we have:
1. Performance
2. Clean up issues
3. Do fixes
4. Streaming, esp. adding cancellation
5. Replacing sax  with saxes

**Comment by alubbe** [comment]
I feel we're making good progress on 1, 4 and 5 - @Siemienik what's your take on 2 & 3?

**Comment by Siemienik** [comment]
ad2. a lot of work what I did gives a result that around 30% current opened issues has labels,
for instance: bug labels: 101 unresolved today

now 427 are open from all 869 issues, I was involved in 192 closed issues.

lots of works we have to do to clear it up

ad3. It relevant enjoyable feeling when I see amazing numbers of a pull request that our fantastic community done. So I am in the best mind for the future :smile:


That I see for now. We should update this list by adding:
6. better typscript integration (some types aren't consistent) 
7. add tests and examples with Angular(some devs has an issue with that)
8. implement a complex worksheet api for ranges operations (styling, filling, values, iterating, etc....)
